### ISSUE TITLE
Italics in model names appearing as tags on landing page, browser tab title

### ISSUE THREAD

**Issue by suzialeksander** [geneontology/noctua#853]
example: gomodel:YeastPathways_PWY-2301 

The imported model name is "<i>myo</i>-inositol biosynthesis - imported from: Saccharomyces Genome Database" but the `<i>` and `</i>` tags remain on the GE browser tab title (FF) as well as the title listing on the Noctua landing page 
![Screenshot 2023-11-30 at 12 07 51](https://github.com/geneontology/noctua/assets/13735608/2447f316-875f-4102-ae6f-061f2e33d7cb)

Name appears as intended in other places 

![Screenshot 2023-11-30 at 12 09 14](https://github.com/geneontology/noctua/assets/13735608/6c32234d-6de7-40b1-a561-f470e56a79b1)

@tmushayahama

**Comment by kltm** [comment]
I think it might be worthwhile to bring this up in a larger context, especially given the recent AGR discussion of the same (thinking yesterday's A-Team call about the same topic). The fact that it works in AmiGO, in this case, is possibly more due to laziness than intent (although it has been a while).

It strikes me as odd that HTML is the "universal" language of making label typesetting (which I assume has its root in paper publication). Is this generally true? Do curators, looking at a file, feel fine about seeing HTML marks for superscript and italic? I would naively assume that ASCII w/o markup would be the preferred base, but I don't know the history here...

**Comment by suzialeksander** [comment]
Personally I'd prefer the text to be readable without tags, but I think we are not shocked when databases fail to handle "special characters".  Even in papers appearing on the publisher&apos;s website it&apos;s not uncommon to have to read through HTML in one way or another. 

 I thought this was a small issue but worth a ticket in case there are other bulk imports, but if there's a bigger story here yes it might be great to bring up somewhere else.

**Comment by kltm** [comment]
I would agree that this, here, is likely an easy fix (just releasing some encoding or stripping tags). We can keep kicking the can down the road on the "bigger" conversation here no problem. That said, I would like to pick this up at some point in the future and figure out what our expectations are for display of names and how they should be output.
